src/vs/workbench/browser/dnd.ts

```diff
@@ -130,6 +137,34 @@ export function extractEditorsDropData(e: DragEvent): Array<IDraggedResourceEdit
 	return editors;
 }
 
+export async function extractTreeDropData(dataTransfer: ITreeDataTransfer): Promise<Array<IDraggedResourceEditorInput>> {
```

Comment:
Fyi unlike `extractEditorsDropData`, this method is actually only being used from inside `ResourcesDropHandler`, so I think we can move it as a private method into there unless you had planned to add tests.

Reply:
Or we remove the `export` and keep it there so that it is closer to the similar method above, as you like.

Reply:
Actually https://github.com/microsoft/vscode/pull/140709#discussion_r789848075 would be related to this.